features/step_definitions/import.ts

```diff
@@ -1,11 +1,24 @@
 import * as assert from "assert";
 import { Given, Then } from "../utils/world";
 import fs from "fs";
+import type { SupportedEncoding } from "../utils/helper";
+import { SUPPORTED_ENCODING } from "../utils/helper";
 
 Given(
   "The csv file {string} with content as below:",
   async function (filePath: string, table) {
-    await this.generateCsvFile(table.raw(), filePath);
+    await this.generateCsvFile(table.raw(), { filePath });
+  },
+);
+
+Given(
+  "The csv file {string} with {string} content as below:",
```

Comment:
It's better to describe what users have to specify on `{string}`

```suggestion
  "The csv file {string} with {string} encoded content as below:",
```

Reply:
Yes. I think the new step definition is better.
It makes the step more clear.